CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to contribute
+
+If you would like to make a contribution to protractor-sync, please check the following steps and guidelines
+ 
+## Installation
+
+Clone the repo
+
+`git clone https://github.com/blackboard/protractor-sync.git`
+
+## Making Changes
+
+Create a branch that summarizes the work you are submitting
+
+`git checkout -b feature/some-protractor-sync-branch`
+
+When adding new features, be sure to add a test to  `protractor-sync/test/spec/protractor_sync_test.ts`
```

Comment:
We actually have more than one test file, so it's not required that they add the tests to this file specifically, but it should be required that they add tests for "new or changed functionality".